### ISSUE TITLE
simplification ?

### ISSUE THREAD

**Issue by ghost** [drolbr/Overpass-API#171]
I guess something like simplification does not exists yet. I didn't found anything in the [language reference](http://wiki.openstreetmap.org/wiki/Overpass_API/Overpass_QL).

To render huge objects as javascript layer in low zoomlevels, this could be a nice accelarting feature.

Maybe

``` c
out simple;
```

or with optional zoomlevel

``` c
out simple 3;
```

**Comment by mmd-osm** [comment]
Doesn't a client side javascript library like http://mourner.github.io/simplify-js/ already address this?

**Comment by jotpe** [comment]
Yes, but it is done on client side. Before, all data must be transferred to the client. Depending on its connection speed it takes some time.
The fun about simplification inside Overpass-API is, that only a small amount of data is send to client. Which leads to quicker responses. Imagine you want to draw the Russian border in zoom level 2. In this case its unnecessary to send the data in highest resolution (ca. 4 MB).

**Comment by joostschouppe** [comment]
For the map I'm trying to build with umap, this would be a great feature. The idea is to show the paved road network at very low zoomlevels. Downloading through overpass and uploading creates data lag and requires downloading lots of data clientside. Linking to an overpass query works (sometimes), but only at high zoomlevels.
